get-started/index.md

```diff
@@ -175,19 +177,22 @@ This message shows that your installation appears to be working correctly.
 List the `hello-world` image that was downloaded to your machine:
 
 ```shell
-$ docker image ls
+docker image ls
 ```
 
 List the `hello-world` container (spawned by the image), which exits after
 displaying its message. If it were still running, you would _not_ need the
 `--all` option:
 
 ```shell
-$ docker container ls --all
+docker container ls --all
+
 CONTAINER ID     IMAGE           COMMAND      CREATED            STATUS
 54f4984ed6a8     hello-world     "/hello"     20 seconds ago     Exited (0) 19 seconds ago
 ```
 
+> If you have docker-ce 1.12 or lower, run `docker ps --all`. For commands introduced in docker-ce 1.13, see [Introducing Docker 1.13](https://blog.docker.com/2017/01/whats-new-in-docker-1-13/){: target="_blank" class="_"}.
```

Comment:
I'm OK with this note, but I think a good assumption to make is that users going through the getting started are just starting with Docker. That means we can assume they have the latest stable version of Docker, and we don't have to put in a bunch of notes explaining things that are relevant for users running an older version.

Reply:
That was my thinking ... I have an idea.